lisa/tools/iperf3.py

```diff
@@ -78,10 +78,7 @@ class Iperf3(Tool):
         r" Gbits/sec.*receiver([\w\W]*?))",
         re.MULTILINE,
     )
-    _json_pattern = re.compile(
-        r"[\w\W]*?(?P<json>{[\w\W]*.*)",
-        re.MULTILINE,
-    )
+    _json_pattern = re.compile(r"\{.*\}", re.DOTALL)
```

Comment:
Add examples above. And from the PR description and commit message, I cannot understand why makes this change.

Reply:
added